packages/three-vrm/src/VRMUtils/combineMorphs.ts

```diff
@@ -108,14 +116,18 @@ export function combineMorphs(vrm: VRMCore): void {
       continue;
     }
 
+    // prevent cloning morph attributes
+    const originalMorphAttributes = mesh.geometry.morphAttributes;
+    mesh.geometry.morphAttributes = {};
+
     const geometry = mesh.geometry.clone();
```

Comment:
Here, cloning takes many CPU resources if the model has many morph attributes. the L121 prevents cloning morph attributes.